src/frontend/screens/Quests/components/QuestDetailsViewPlay/index.tsx

```diff
@@ -33,6 +34,21 @@ export function QuestDetailsViewPlayWrapper({
     questMeta?.eligibility?.steam_games ?? []
   )
 
+  const navigateToGame = useMutation({
+    mutationFn: async (appName: string) => {
+      await window.api.addHyperplayGame(appName)
+      const gameInfo = await getGameInfo(appName, 'hyperplay')
+      navigate(`/gamepage/hyperplay/${appName}`, {
+        state: { gameInfo, fromDM: false }
+      })
+    },
+    onError: (error, variable) => {
+      window.api.logError(
+        `Error navigating to ${variable} game: ${error.message}`
+      )
+    }
+  })
+
```

Comment:
I made this a mutation to have disable the CTA button while loading, adding the game to the library can take up to 3 seconds